src/AcceptanceTests/When_using_token_credential_with_fully_qualified_namespace.cs

```diff
@@ -0,0 +1,87 @@
+namespace NServiceBus.Transport.AzureServiceBus.AcceptanceTests
+{
+    using System;
+    using System.Threading.Tasks;
+    using AcceptanceTesting;
+    using Azure.Identity;
+    using Azure.Messaging.ServiceBus;
+    using NServiceBus.AcceptanceTests;
+    using NServiceBus.AcceptanceTests.EndpointTemplates;
+    using NUnit.Framework;
+
+    public class When_using_token_credential_with_fully_qualified_namespace : NServiceBusAcceptanceTest
+    {
+        string fullyQualifiedNamespace;
+
+        [SetUp]
+        public void Setup()
+        {
+            var connectionString = Environment.GetEnvironmentVariable("AzureServiceBus_ConnectionString");
+            var connectionStringProperties = ServiceBusConnectionStringProperties.Parse(connectionString);
+            fullyQualifiedNamespace = connectionStringProperties.FullyQualifiedNamespace;
+        }
+
+        [Test]
+        public async Task Should_work()
+        {
+            var context = await Scenario.Define<Context>()
+                .WithEndpoint<Publisher>(b =>
+                {
+                    b.CustomConfig(c =>
+                    {
+                        var transport = c.ConfigureTransport<AzureServiceBusTransport>();
```

Comment:
Does our test framework not allow the new API to be used? (`c.ConfigureTransport(new AzureServiceBusTransport(fqns, credential))`

Reply:
Yeah I will fix this

Reply:
Actually, I think we have to use that one; otherwise we are essentially setting a new transport definition and would not get the default values from the template if I recall correctly